### DISCUSSION TITLE
Wanting to help with maintenance, headless fork, and other contributions

### DISCUSSION THREAD

**Discussion by sygint** [CKGrafico/papanasi#86] [Q&A] [answered]
Hey, I love mitosis and I saw this project as really the only example of a component library, but I've been thinking a lot about styling and want to potentially make a fork based on headless UI, so just the behaviors, theming separate (or maybe optional?). I'd love to talk to you and pick your brain on that as well as potentially help you with maintaining this library, or maybe just maintaining a fork and upstreaming to both Papanasi and my headless version. Anyhow, I'd love to chat!

**Comment by sygint** [comment]
Upon further investigation, it seems you actually have a headless version implemented? So if that's the case, I don't even need a fork, but maybe I can help with the design, testing, and maybe accessiblity

**Reply by CKGrafico** [reply, marked as answer]
Hello yes, papanasi is ready to be theamable, asi is explained here https://papanasi.js.org/?path=/docs/%F0%9F%93%97documentation-customization--page#themes

I'm a little bit busy working on other things then I'm totally open for PRs for sure :)
